scipy/fft/_basic_backend.py

```diff
@@ -0,0 +1,183 @@
+from scipy._lib._array_api import array_namespace, is_numpy
+from . import _pocketfft
+import numpy as np
+
+
+def arg_err_msg(param):
+    return f'Providing {param!r} is only supported for numpy arrays.'
+
+
+def _validate_fft_args(overwrite_x, workers, plan, norm):
+    if overwrite_x:
+        raise ValueError(arg_err_msg("overwrite_x"))
```

Comment:
`overwrite_x` is just a hint so it should be fine to drop this argument when calling other backends.

Reply:
Okay, happy to change this.

Reply:
Addressed https://github.com/scipy/scipy/pull/19005/commits/ea36a042f2f090c2aca20d80509b889aa7808bea.